packages/fxa-payments-server/src/components/PaymentMethodHeader/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Localized } from '@fluent/react';
+import { Plan } from 'fxa-shared/subscriptions/types';
+import React from 'react';
+import useValidatorState from '../../lib/validator';
+import { Form } from '../fields';
+import { PaymentConsentCheckbox } from '../PaymentConsentCheckbox';
+
+export type PaymentMethodHeaderProps = {
+  plan: Plan;
+  onClick: (event: React.MouseEvent<HTMLInputElement>) => void;
+  prefix?: string;
+};
+
+export const PaymentMethodHeader = ({
+  plan,
+  onClick,
+  prefix = '',
+}: PaymentMethodHeaderProps) => {
+  const checkboxValidator = useValidatorState();
+  const headerMarginTop = '40px';
+  return (
+    <>
+      {prefix ? (
+        <Localized id="payment-method-header-prefix" vars={{ prefix }}>
+          <h2
+            style={{ marginTop: headerMarginTop }}
+            data-testid="header-prefix"
+          >
+            Choose your payment method
```

Comment:
`prefix` isn't used in this fallback string.

Reply:
Ah good spot, added the prefix to the fallback.